### ISSUE TITLE
Dashboard Widgets gone after Version 2.2.0  update

### ISSUE THREAD

**Issue by bomshteyn** [beyondcode/invoker-community#107]
All the Dashboard Widgets are gone after updating to Version 2.2.0

Model widgets still there.

**Comment by bomshteyn** [comment]
Same issue with 2.3 no ability to save persist dashboard widgets, any plan to fix this?

**Comment by mechelon** [comment]
Can you reproduce this error with the latest version, 2.7.0?

**Comment by bomshteyn** [comment]
Yes, still same issue. The widget on the model page persists. But the dashboard widgets are gone as soon as i close the project in the invoker.

**Comment by mechelon** [comment]
Can you share the following information with us:
- remote / local project
- PHP Version
- Laravel Version
- OS Version
Thank you!

**Comment by bomshteyn** [comment]
- remote
- PHP 7.4
- Laravel v8.56.0
- Ubuntu 20.04 (LTS) x64
